crypto/asn1/a_gentm.c

```diff
@@ -163,56 +223,49 @@ ASN1_GENERALIZEDTIME *ASN1_GENERALIZEDTIME_set(ASN1_GENERALIZEDTIME *s,
     return ASN1_GENERALIZEDTIME_adj(s, t, 0, 0);
 }
 
+int ASN1_GENERALIZEDTIME_cmp_time_t(const ASN1_GENERALIZEDTIME *s, time_t t)
```

Comment:
dislike this function, who needs to compare ASN1 to time_t?
To compare ASN1 times, strcmp (internally) is fine.